contexts/arbo-context/filter-update-steps/add-action-to-selected-filters-step.tsx

```diff
@@ -0,0 +1,18 @@
+import { HandleArboFilterUpdateInput, HandleArboFilterUpdateOutput } from "../arbo-filter-update-steps";
+
+export const addActionToSelectedFiltersStep = (
```

Comment:
Doing this step first isn't the old way of doing it but it makes the other steps simpler (see `adjustMapPositionStep`)